features/steps/steps.go

```diff
@@ -218,3 +225,35 @@ func (c *Component) theFollowingExportStartEventsAreProduced(events *godog.Table
 	}
 	return nil
 }
+
+func (c *Component) theGeographyDatsetJSONResult(expected *godog.DocString) error {
```

Comment:
Is there not a built in comparison function in the api feature for checking this result?

Reply:
The built-in function does a straight compare of two pieces of json, whereas I want to selectively compare, in particular when comparing urls, to compare paths and ignore certain hostnames